bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "exceptionless",
-  "version": "0.9.1",
+  "version": "0.9.2",
```

Comment:
We don't need to bump it to 0.9.2 because we haven't pushed 0.9.1 :). I can fix this tho.